frame/treasury/src/lib.rs

```diff
@@ -156,6 +169,15 @@ pub trait Trait: frame_system::Trait {
 
 	/// Percentage of spare funds (if any) that are burnt per spend period.
 	type Burn: Get<Permill>;
+
+	/// The amount held on deposit for placing a bounty proposal.
+	type BountyDepositBase: Get<BalanceOf<Self>>;
```

Comment:
probably no pont having this be different from `TipReportDepositBase`. just amalgamate them both into `DataDepositBase`